src/types/cart.d.ts

```diff
@@ -45,21 +45,9 @@ export interface Cart {
   links?: {}
   meta?: {
     display_price?: {
-      with_tax?: {
-        amount: number
-        currency: string
-        formatted: string
-      }
-      display_price?: {
-        amount: number
-        currency: string
-        formatted: string
-        tax: {
-          amount: number
-          currency: string
-          formatted: string
-        }
-      }
+      with_tax?: FormattedPrice
+      without_tax?: FormattedPrice
+      tax?: FormattedPrice
       timestamps?: {
```

Comment:
@mitchellbutler I think you may have dropped a `}` here - `timestamps` should be nested under `meta` rather than `display_price`

Reply:
Oops, you're right. I've just added a commit which should resolve this.